var/spack/repos/builtin/packages/hdf5-vol-external-passthrough/package.py

```diff
@@ -0,0 +1,25 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Hdf5VolExternalPassthrough(CMakePackage):
+    """Package for HDF5 external pass-through VOL."""
+
+    homepage = "https://sdm.lbl.gov/"
+    git      = "https://github.com/hpc-io/vol-external-passthrough.git"
+    maintainers = ['hyoklee']
+
+    version('v1.0')
+    depends_on('hdf5@develop-1.13')
+
+    def cmake_args(self):
+        """Populate cmake arguments for HDF5 DAOS."""
```

Comment:
Is "DAOS" a typo?

Reply:
@lrknox , please remove the comment line.